pkg/configurator/types.go

```diff
@@ -22,6 +22,14 @@ type Client struct {
 	cacheSynced      chan interface{}
 }
 
+// CRDClient is the k8s client struct for the MeshConfig CRD. The feature is in experimental stage.
+type CRDClient struct {
```

Comment:
I wonder if this has to be exported (capitalized).
It seems to me that, so far, this is only used in `pkg/configurator`, which means that this could be `crdClient`.
I know lowercasing this would conflict with another `crdClient`, but that could be renamed to `client` perhaps.

Reply:
SGTM. We can rename `CRDClient` to `client` when the current `client` is removed.